src/main/java/uk/ac/ebi/eva/evaseqcol/entities/SeqColExtendedDataEntity.java

```diff
@@ -164,10 +166,10 @@ public static SeqColExtendedDataEntity constructSeqColSortedNameLengthPairs(
         // Get the sorted list
         List<String> sortedNameLengthPairsList = constructSortedNameLengthPairs(nameLengthPairList);
```

Comment:
Think you missed the one in `constructSortedNameLengthPairs` (lines 192+), it's still `SHA512ChecksumCalculator` rather than `DigestCalculator`

Reply:
True, I'll fix it